### PR TITLE
add December 2022 meeting page

### PR DESCRIPTION
**Pull request by jabawack81**

### CODE REVIEW

**Comment by fcheung** [comment]
There’s also Daniel’s talk (on queues)

File: source/meetings/2022/december/index.html.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+December*, from _6:00pm_ to _8:00pm_ (meeting starts at _6:30pm_).
+
+**👪 in person meeting alert 👪**
+
+and with the December meetup it will be our fourth in person meetup of the 2022; all thanks to the helps of the amazing people at [Zinc](https://www.zinc.vc) at [their offices][zinc-venue], on Eversholt Street. [Full venue and
+registration details are given below](#dec22registration).
+
+## Agenda
+
+### Mutation testing - study case"
```
Comment (fcheung) [inline source/meetings/2022/december/index.html.md:28]: There’s a stray “ here

File: source/meetings/2022/december/index.html.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Hopefully you all remember that physical meetings involve finite space and so to be guaranteed entry **[you need to register via eventbrite][december2022-eventbrite]**.
+
+### Venue
+
+The address of the venue:
+
+> Euston House, 24 Eversholt Street<br/>London, NW1 1AD<br/>England, United Kingdom[See on a map][zinc-venue]
+
+The venue has a hard limit of 70 people.  If you register and realise you
```
Comment (fcheung) [inline source/meetings/2022/december/index.html.md:74]: Their survey response says 100 people, unless I’ve missed an email somewhere

Comment (jabawack81) [reply source/meetings/2022/december/index.html.md]: 👍

File: source/meetings/2022/december/index.html.md (hunk 1 above)
Comment (jabawack81) [reply source/meetings/2022/december/index.html.md]: 👍

**Comment by jabawack81** [comment]
@fcheung fixed all, I thought Daniel was been moved but I was wrong, thanks

**Comment by fcheung** [comment]
Yeah there was a lot of back and forth earlier this week with  changes to the lineup.

File: source/meetings/2022/december/index.html.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+> result. In the end, it all boils down to one thing: keeping latency low. In
+> this talk I will present a latency-focused approach to managing your queues
+> reliably, keeping your jobs flowing and your users happy. 
+
+## Afterwards
+
+When the talks come to an end we'll decamp to a local pub for some food, some
+drinks and some chat with your fellow attendees.
+
+## Venue & Registration {#dec22registration}
```
Comment (fcheung) [inline source/meetings/2022/december/index.html.md:70]: Is the right anchor syntax? I’m hopeless with markdown links but this looks weird on the GitHub preview

File: source/meetings/2022/december/index.html.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+Prior to attending you should familiarise yourself with our
+[README](http://readme.lrug.org/) paying close attention to [the code of
+conduct](http://readme.lrug.org/#code-of-conduct) which applies to all
+attendees, even though we are all in our own little bubbles.
```
Comment (fcheung) [inline source/meetings/2022/december/index.html.md:75]: I think we might be dropping the “our own little bubbles bit” now that we’re back in person - I think we removed that phrase from a few places recently

File: source/meetings/2022/december/index.html.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+> efficiency, and then we'll talk about how our implementation broke
+> Garbage Collection.
+
+> After that we'll go on a journey together, through some weeds, and
+> taking a few bad turns until we finally emerge with a few PR's that
+> not only Fix GC, but make our project better too.
+
+###  What does "high priority" mean? The secret to happy queues
+
+[Daniel Magliola]()
```
Comment (fcheung) [inline source/meetings/2022/december/index.html.md:54]: Maybe use the same Twitter link as went he last spoke? https://mobile.twitter.com/dmagliola

**Review by fcheung — APPROVED** [review]
I left some extra small comments but all the important stuff looks good!

File: source/meetings/2022/december/index.html.md (hunk 4 above)
Comment (h-lame) [reply source/meetings/2022/december/index.html.md:75]: Yup - we also added a reminder about the code of conduct to the afterwards part.  You can copy the changes we made to the November meeting in [this PR](https://github.com/lrug/lrug.org/pull/245/files#diff-e8429d7adff1fae58f442f383bc74f25fd2079ff6d74af0d3555a970cdccfb9d).

File: source/meetings/2022/december/index.html.md (hunk 3 above)
Comment (jabawack81) [reply source/meetings/2022/december/index.html.md:70]: it should be correct is the same as other pages

File: source/meetings/2022/december/index.html.md (hunk 4 above)
Comment (jabawack81) [reply source/meetings/2022/december/index.html.md:75]: 👍

File: source/meetings/2022/december/index.html.md (hunk 5 above)
Comment (jabawack81) [reply source/meetings/2022/december/index.html.md:54]: I looked around and didn't found it and forgot about that

File: source/meetings/2022/december/index.html.md (hunk 3 above)
Comment (h-lame) [reply source/meetings/2022/december/index.html.md:70]: I think the issue here is that the GitHub preview isn't using exactly the same markdown renderer as the site so it's never gonna look exactly right in the preview.

**Pull request merged**